### PR TITLE
remove keyboard helper screen from web chat

### PR DESCRIPTION
**Pull request by amirmursal**
> Fix candidate for #5220, #5212, #4965, #4918

## Changelog Entry

Fixes [#4965](https://github.com/microsoft/BotFramework-WebChat/issues/4965). Removed keyboard helper screen in [#5234](https://github.com/microsoft/BotFramework-WebChat/pull/5234), by [@amirmursal](https://github.com/amirmursal)

## Description

Removed keyboard control screen from the chat window to address several accessibility issues

-  [x] I have added tests and executed them locally
-  [x] I have updated `CHANGELOG.md`
-  [x] I have updated documentation

## Review Checklist

> This section is for contributors to review your work.

-  [x] Accessibility reviewed (tab order, content readability, alt text, color contrast)
-  [x] ~Browser and platform compatibilities reviewed~
-  [x] ~CSS styles reviewed (minimal rules, no `z-index`)~
-  [x] Documents reviewed (docs, samples, live demo)
-  [x] ~Internationalization reviewed (strings, unit formatting)~
-  [x] ~`package.json` and `package-lock.json` reviewed~
-  [x] ~Security reviewed (no data URIs, check for nonce leak)~
-  [x] Tests reviewed (coverage, legitimacy)

### CODE REVIEW

**Review by beyackle2 — APPROVED** [review]

File: __tests__/html/transcript.navigation.visualKeyboardIndicator.html [hunk 1]
```diff
@@ -52,36 +52,7 @@
...
         await host.snapshot();
-
-        // WHEN: SHIFT-TAB key is pressed from the transcript.
-        await host.sendShiftTab();
-
-        // THEN: It should show the keyboard help.
-        await host.snapshot();
-
-        // WHEN: SHIFT-TAB key is pressed from the keyboard help.
-        await host.sendShiftTab();
```
Comment (OEvgeny) [inline __tests__/html/transcript.navigation.visualKeyboardIndicator.html:63]: I think you should remove only these two lines, everything bellow should work as before. May also need to update snapshots to reflect changes.

**Review by OEvgeny — CHANGES_REQUESTED** [review]
Please update tests according to my comment

File: __tests__/html/transcript.navigation.visualKeyboardIndicator.html [hunk 2]
```diff
@@ -52,8 +52,31 @@
...
+        await host.sendShiftTab();
+
+        // THEN: It should focus on the first button outside of Web Chat.
+        await host.snapshot();
+
+        // WHEN: TAB key is pressed from the widget before Web Chat.
+        await host.sendTab();
+
+        // THEN: It should show the keyboard help.
+        await host.snapshot();
```
Comment (OEvgeny) [inline __tests__/html/transcript.navigation.visualKeyboardIndicator.html:66]: This step should be removed as we remove the screen

Comment (OEvgeny) [reply __tests__/html/transcript.navigation.visualKeyboardIndicator.html:66]: The rest of the test should work as before

**Review by OEvgeny — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -31,7 +31,7 @@ Notes: web developers are advised to use [`~` (tilde range)](https://github.com/
    -  Added `DecoratorComposer` and `ActivityDecorator` to be used for decorating activity border, in PR [#5205](https://github.com/microsoft/BotFramework-WebChat/pull/5205), by [@OEvgeny](https://github.com/OEvgeny)
 
 ### Fixed
-
+-Fixes [#4965](https://github.com/microsoft/BotFramework-WebChat/issues/4965). Removed keyboard helper screen in [#5234]https://github.com/microsoft/BotFramework-WebChat/pull/5234), by [@amirmursal](https://github.com/amirmursal)
```
Comment (OEvgeny) [inline CHANGELOG.md:34]:
```suggestion
-  Fixes [#4965](https://github.com/microsoft/BotFramework-WebChat/issues/4965). Removed keyboard helper screen in [#5234]https://github.com/microsoft/BotFramework-WebChat/pull/5234), by [@amirmursal](https://github.com/amirmursal)
```

**Comment by OEvgeny** [comment]
We probably want to cleanup localizations as well:
https://github.com/microsoft/BotFramework-WebChat/blob/e069c6258cce746d27f540858ef62f773baa45c5/packages/api/src/localization/en-US.json#L74-L90

**Comment by OEvgeny** [comment]
> We probably want to cleanup localizations as well:
> 
> https://github.com/microsoft/BotFramework-WebChat/blob/e069c6258cce746d27f540858ef62f773baa45c5/packages/api/src/localization/en-US.json#L74-L90
Done

**Review by compulim — APPROVED** [review]

**Comment by OEvgeny** [comment]
Closes #4995
Closes #4996
Closes #4997
as well

**Pull request merged**